tests/test_blob_ndjson_format.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import json
+from azure.storage.blob import BlobServiceClient
+from azure.core.exceptions import ResourceNotFoundError
+
+
+class TestAzureBlobNDJSONFormat(unittest.TestCase):
```

Comment:
don't extend unittest.TestCase, if it requires external integration

Reply:
done